test/vault/secrets/logical_test.clj

```diff
@@ -0,0 +1,108 @@
+(ns vault.secrets.logical-test
+  (:require
+    [clojure.test :refer [is testing deftest]]
+    [vault.client.http]
+    [vault.secrets.logical :as vault-logical])
```

Comment:
Now that the basic "secret client" protocol is implemented by the clients, i think this "logical" secrets engine should be renamed "kvv1" to match actually naming in vault.

Reply:
Good point :) agreed